goreleaserlib/goreleaser.go

```diff
@@ -21,25 +21,27 @@ import (
 	"github.com/goreleaser/goreleaser/pipeline/env"
 	"github.com/goreleaser/goreleaser/pipeline/fpm"
 	"github.com/goreleaser/goreleaser/pipeline/git"
+	"github.com/goreleaser/goreleaser/pipeline/gpg_signing"
```

Comment:
https://blog.golang.org/package-names

having underscores in package names is not a good practice..